generators/server/templates/dotnetcore/src/Project/Infrastructure/SwaggerStartup.cs.ejs

```diff
@@ -14,14 +14,15 @@
 -%>
 using Microsoft.AspNetCore.Builder;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.OpenApi.Models;
 using Swashbuckle.AspNetCore.Swagger;
 
 namespace <%= namespace %>.Infrastructure {
     public static class SwaggerConfiguration {
         public static IServiceCollection AddSwaggerModule(this IServiceCollection @this)
         {
             @this.AddSwaggerGen(c => {
-                c.SwaggerDoc("v2", new Info {Title = "nhipsterSampleApplication API", Version = "0.0.1"});
+                c.SwaggerDoc("v2", new OpenApiInfo {Title = "nhipsterSampleApplication API", Version = "0.0.1"});
```

Comment:
Can you take the opportunity to replace `nhipsterSampleApplication` by the application name variable please.

Reply:
Good idea